### PR TITLE
Move filter skipped messages to debug level, reduce chatter on busy f…

### PR DESCRIPTION
**Pull request by jlrgraham**
…ilters.

I'd propose decreasing the severity of this log message.  I've got a filter setup that is skipping messages on a relatively busy broker and I'm winding up with logs that are mainly filter skip messages.

I'd also considered trying to make this a configurable option; maybe adding a `config.skipped_messages_log_level` setting, but wanted to propose here first.  If that's preferred please let me know and I'd update the PR here.  Thanks!

### CODE REVIEW

**Comment by jpmens** [comment]
I think making this configurable is a good idea, and we’ll gladly take your updated patch.

File: tests/test_core_main.py [hunk 1]
```diff
@@ -128,6 +128,29 @@ def test_message_log(caplog):
...
+    """
+    Verify that setting the filteredmessagesloglevel config option changes the log level of the "Filter in section" message.
+    """
+
+    # Bootstrap the core machinery without MQTT.
+    core_bootstrap(configfile=configfile_full)
+
+    # Adjust `filteredmessagesloglevel` configuration setting.
+    mocker.patch("mqttwarn.core.cf.filteredmessagesloglevel", 'DEBUG')
+    mocker.patch("mqttwarn.core.cf.filteredmessagesloglevelnumber", 10)
```
Comment (jlrgraham) [inline tests/test_core_main.py:142]: I wasn't really sure how to handle this here.  Optimally it seems like we'd set the `filteredmessagesloglevel` string and then this test should cover the config object evaluating that and converting it internally to the `filteredmessagesloglevelnumber` value, but the mock patches here are applied after the configuration object is initialized.  Suggestions?

**Comment by jpmens** [comment]
Thank you for submitting this.

@amotl could you please chime in regarding the testing stuff and @jlrgraham's question?

**Comment by amotl** [comment]
Hi, and thanks for your efforts. I'm currently afk, but may be able to have a look later this evening. Cheers, Andreas.

On 13 November 2022 11:14:05 CET, Jan-Piet Mens ***@***.***> wrote:
>Thank you for submitting this.
>
***@***.*** could you please chime in regarding the testing stuff and @jlrgraham's question?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/jpmens/mqttwarn/pull/608#issuecomment-1312693864
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
-- 
Sent from my mind. This might have been typed on a mobile device, so please excuse my brevity.

**Review by jpmens — APPROVED** [review]

**Comment by amotl** [comment]
Hi again,

apologies for the delay. It looks you figured out to come up with a good test case. Great! I think it will be ready to merge, unless @jpmens or @sumnerboy12 think a bit of documentation could be added?

With kind regards,
Andreas.

**Comment by jpmens** [comment]
No, it's fine as it is; the parameter is well documented in the sample INI files.

Thank you, @jlrgraham, for  your work!

**Comment by jpmens** [comment]
>Ah, I missed the change in `HANDBOOK.md`. Excellent. Merging.

I beat you to it! ;-)

**Comment by amotl** [comment]
> the parameter is well documented in the sample INI files.

The sample INI files are actually only configuration files provided for testing purposes, many users will not see them. Also, `mqttwarn make-config` would use the file at [1]. I will not nitpick about it, just wanted to outline that the new setting could also be added there in order to increase visibility.

[1] https://github.com/jpmens/mqttwarn/blob/main/mqttwarn/examples/basic/mqttwarn.ini

**Pull request merged**